mozjexl/lib/Lexer.js

```diff
@@ -67,10 +69,13 @@ Lexer.prototype.getTokens = function(elements) {
 	var tokens = [],
 		negate = false;
 	for (var i = 0; i < elements.length; i++) {
+
```

Comment:
Super nit: this newline is probably not needed. That being said, all of mozjexl needs a style pass, so I wouldn't stress much about it.